react-first-app/src/app/components/App/Board/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+
+import logo from '../../../../logo.svg';
+
+import style from './styles.css';
+
+import Square from '../Square';
+
+class Board extends React.Component {
+  renderSquare(i) {
+    return (
+      <Square
+        value={this.props.squares[i]}
+        onClick={() => this.props.onClick(i)}
```

Comment:
We avoid inline functions as a new one is created each time the render function is called. To fix this:
```
handleClick = i => this.props.onClick(i)

<Square value={this.props.squares[i]} onClick={this.handleClick(i)}>
```